docs/RELEASE.md

```diff
@@ -26,18 +26,55 @@ docker pull registry.scality.com/cloudserver/cloudserver:<tag>
 
 To release a production image:
 
-* Name the tag for the repository and Docker image.
+* Checkout the relevant branch. In this example,
+  we are working on development/8.3, and we want to release version `8.3.0`.
 
-* Use the `yarn version` command with the same tag to update `package.json`.
+```sh
+git checkout development/8.3
+```
 
-* Create a PR and merge the `package.json` change.
+* Tag the branch with the release version. In this example, `8.3.0`
 
-* Tag the repository using the same tag.
+```sh
+git tag -a 8.3.0
+# The message should be 'v<version>'
+v8.3.0
+```
+
+* Push the tags to GitHub.
+
+```sh
+git push --tags
+```
+
+* With the following parameters, [force a build here](https://eve.devsca.com/github/scality/cloudserver/#/builders/3/force/force)
+
+    * Branch Name: The one used for the tag earlier. In this example 'development/8.3'
+    * Override Stage: 'release'
+    * Extra properties:
+      * name: `'tag'`, value: `[release version]`, in this example`'8.3.0'`
+
+* Once the docker image is present on [registry.scality.com](registry.scality.com),
+  update Vault's `package.json`
```

Comment:
It's cloudserver here! :smile:

Reply:
Yep, great idea! thanks

Reply:
You forgot to change `Vault` here :slightly_smiling_face: